models/product-discount/src/product-discount-draft/transformers.ts

```diff
@@ -0,0 +1,31 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type {
+  TProductDiscountDraft,
+  TProductDiscountDraftGraphql,
+} from '../types';
+
+const transformers = {
+  default: Transformer<TProductDiscountDraft, TProductDiscountDraft>(
+    'default',
+    { buildFields: ['value', 'name', 'description'] }
+  ),
+  rest: Transformer<TProductDiscountDraft, TProductDiscountDraft>('rest', {
+    buildFields: ['value', 'name', 'description'],
+  }),
+  graphql: Transformer<TProductDiscountDraft, TProductDiscountDraftGraphql>(
+    'graphql',
+    {
+      buildFields: ['value', 'name', 'description'],
+      addFields: ({ fields }) => ({
+        nameAllLocales: LocalizedString.toLocalizedField(fields.name),
+        descriptionAllLocales: LocalizedString.toLocalizedField(
+          fields.description
+        ),
+        __typename: 'ProductDiscountDraft',
+      }),
```

Comment:
I don't see these fields in the GraphQL docs

![image](https://user-images.githubusercontent.com/8136478/217640962-30e42f1f-e274-4a13-a4d8-a129ea51e0bb.png)

Reply:
Whoooooopsie! Thanks for checking so thoroughly!

Reply:
Resolved in 7dde2eedf72087d87efe2186104a3a1b12eab5fb